nuget/lib/dependabot/nuget/update_checker/version_finder.rb

```diff
@@ -83,7 +83,9 @@ def find_compatible_version(sorted_versions)
           return first_version unless version_compatible?(dependency.version)
           return first_version if version_compatible?(first_version.fetch(:version))
```

Comment:
This line can be removed as it will now be the run twice as part of the `.find(`

Reply:
Good catch; fixed.